evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -136,6 +142,20 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
         };
     </script>
 
+    {% trans 'Replace participants' as title %}
+    {% blocktrans asvar question %}Do you really want to delete all existing participants and replace them with participant data from the Excel file?{% endblocktrans %}
+    {% trans 'Replace participants' as action_text %}
+    {% include 'confirmation_modal.html' with modal_id='replaceParticipantsModal' title=title question=question action_text=action_text btn_type='danger' %}
+    <script type="text/javascript">
+        function replaceParticipantsModalAction(dataId) {
+            var input = $("<input>")
+            .attr("type", "hidden")
+            .attr("name", "operation").val(dataId);
```

Comment:
personally, I'd indent these two lines, but I can also live with how it is now.

Reply:
You just picked them as an example, but would intend them at every button?

Reply:
Uh, good point, I'd make an extra commit that "fixes" the indentation for this and the other 6 spots where it is not indented.